modular_splurt/code/game/object/items/implants/implant_gfluid.dm

```diff
@@ -77,3 +79,15 @@
 	name = "implant case - 'Genital Fluid'"
 	desc = "A glass case containing a Genital FLuid Infuser implant."
 	imp_type = /obj/item/implant/genital_fluid
+
+/obj/item/implantcase/genital_fluid/emag_act(mob/user)
+	. = ..()
+	if(istype(imp, /obj/item/implant/genital_fluid))
+		var/obj/item/implant/genital_fluid/I = imp
+		if(I.use_blacklist == FALSE) // already hacked
```

Comment:
Unless there's  good reason not to I'd keep it consistent with other instances of emag_act and set the EMAGGED flag in obj_flags.